app/containers/CallHeader.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import Touchable from 'react-native-platform-touchable';
+
+import { useAppSelector } from '../lib/hooks';
+import { useTheme } from '../theme';
+import sharedStyles from '../views/Styles';
+import { CustomIcon } from './CustomIcon';
+import { BUTTON_HIT_SLOP } from './message/utils';
+import AvatarContainer from './Avatar';
+import StatusContainer from './Status';
+import DotsLoader from './DotsLoader';
+
+type TCallHeader = {
+	mic: boolean;
+	cam: boolean;
+	setCam: Function;
+	setMic: Function;
+	title: string;
+	avatar: string;
+	uid: string;
+	name: string;
+	direct: boolean;
+};
+
+export const CallHeader = ({ mic, cam, setCam, setMic, title, avatar, uid, name, direct }: TCallHeader): React.ReactElement => {
+	const style = useStyle();
+	const { colors } = useTheme();
+	const calling = useAppSelector(state => state.videoConf.calling);
+
+	const handleColors = (enabled: boolean) => {
+		if (calling) {
+			if (enabled) return { button: colors.conferenceCallCallBackButton, icon: colors.gray300 };
+			return { button: 'transparent', icon: colors.gray100 };
+		}
+		if (enabled) return { button: colors.conferenceCallEnabledIconBackground, icon: colors.conferenceCallEnabledIcon };
+		return { button: 'transparent', icon: colors.conferenceCallDisabledIcon };
+	};
+
+	return (
+		<View>
+			<View style={style.actionSheetHeader}>
+				<View style={style.rowContainer}>
+					<Text style={style.actionSheetHeaderTitle}>{title}</Text>
+					{calling ? <DotsLoader /> : null}
+				</View>
+				<View style={style.actionSheetHeaderButtons}>
```

Comment:
Should these options be persisted for next call or they're just default?

Reply:
default